tf2onnx/tfonnx.py

```diff
@@ -1267,6 +1260,11 @@ def minmax_op(ctx, node, name, args):
 
 
 def pack_op(ctx, node, name, args):
+    # in tf, "pack" can accept one input tensor which means doing nothing,
+    # so remove the node in ONNX
+    if len(node.inputs) == 1:
+        ctx.replace_all_inputs(ctx.get_nodes(), node.output[0], node.input[0])
+        return None
     # hack to make up for the missing onnx pack op
```

Comment:
[optional] personally, i prefer to insert a new line here.

Reply:
agree, showing that this is a code path for special case.